package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "mdlinks",
+  "version": "1.0.0",
+  "description": "Proyecto mdlinks de Laboratoria",
```

Mejorar la descripción en ingles
```suggestion
  "description": "Proyecto mdlinks de Laboratoria",
```